nbi/buildspec.yml

```diff
@@ -20,13 +20,14 @@ phases:
       - echo "Setting up conda"
       - export PATH=$LCC_CONDA_INSTALL_DIR/bin:$PATH
       - for CHANNEL in conda-forge cf-staging; do conda config --system --append channels $CHANNEL; done
-      - conda install --yes conda-pack
+      - conda install --force-reinstall --yes conda-pack=0.7.0
```

Comment:
nit: curious, force-reinstall necessary, since we are pinning the conda-pack version?

Reply:
It is slightly more time but due to issues with the site-package in `0.6.0` for Python  3.10, I wanted to ensure that the previous version was entirely gone.